### ISSUE TITLE
[FR] Log filename should end with .log

### ISSUE THREAD

**Issue by henri9813** [AppFlowy-IO/AppFlowy#7008]
### Description

Hello,

currently, my log file has as extension  `date`. 

and it should be in a dedicated directory

`logs/${YYYY-mm-dd}.log`

### Impact

more easy to find log and open them.

### Additional Context

# Screenshot

<img width="748" alt="image" src="https://github.com/user-attachments/assets/331c1a27-fb2d-463c-9843-71a4c89177ee" />

**Comment by LucasXu0** [comment]
The export log file logic also needs to be updated.

**Comment by LucasXu0** [comment]
@henri9813 Yes. Here is the [guide](https://appflowy.com/ccf3ca5a-7ecd-4c35-805c-3a3c9c22e87d/AppFlowy-Q-A) for exporting the log files.

**Comment by henri9813** [comment]
@annieappflowy why did you close the issue ?

**Comment by annieappflowy** [comment]
Won't work on this request
